finesse/config.py

```diff
@@ -76,6 +76,18 @@
 a request.
 """
 
+FTSW500_HOST = "127.0.0.1"
+"""The IP address or hostname of the machine running the FTSW500 software."""
+
+FTSW500_PORT = 7778
+"""The port on which the TCP server of FTSW500 is listening."""
+
+FTSW500_POLLING_INTERVAL = 1.0
+"""How long to wait between polls of the EM27's status."""
```

Comment:
Is this related to `EM27_SENSORS_POLL_INTERVAL` above? Could you just use that instead? Or maybe is `EM27` a typo?

Reply:
Good spot 😅